spec/features/admin_adds_new_video_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.